### PR TITLE
Conference website pull request

### PR DESCRIPTION
**Pull request by codecaiine**
- I created a new repository for the capstone project
- I created a new branch for mobile and desktop
- I set up linters for HTML CSS & Javascript
- I added Javascript Scripts to create speakers section Dynamically
- I added descriptive readme file
- I fixed linters errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/codecaiine/conference-website/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (abruzy) [inline README.md:62]: kindly give credit to [Cindy Shin](https://www.behance.net/adagio07) the [original author of the design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as required by the [Creative Common Licence](https://creativecommons.org/licenses/)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        </div>
+    </section>
+    <section class="section-speaker" id="my_speakers">
+        <div class="container">
+            <div class="main-title">
+                <h1 class="title">Featured Speakers</h1>
+                <div class="title-line"></div>
+            </div>
+
+            <div class="row mb-2" id="speakers-list"></div>
```
Comment (abruzy) [inline index.html]: your featured speaker's sections should have two columns of three articles each

Screenshot 2021-08-12 at 20 36 29

Here's yours, you have 4

Screenshot 2021-08-12 at 20 37 19

So please fix this

File: index.html [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+                <div class="col-lg-10 col-md-6">
+                    <div class="footer-copy-right text-center">
+
+                        <h4>Copyright © 2021 All rights reserved | This website is made with <i class="fa fa-heart color-danger" aria-hidden="true"></i> with <a href="https://getbootstrap.com/">bootstrap .</a></h4>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
```
Comment (abruzy) [inline index.html]: follow semantic HTML, `footer` should have the `footer` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+                    </ul>
+                    <ul class="link-info">
+                        <li>French</li>
+                        <li>Sign Up</li>
+                        <li>Login</li>
+                    </ul>
+
+                </div>
+            </div>
+        </div>
```
Comment (abruzy) [inline index.html]: use Semantic HTML, this is still a `nav` tag, the only difference is we have two navbars one is the top nav and the other is bottom nav, so change the div to `nav` tag

Making use of semantic HTML is very good as this helps in improving your application SEO

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @codecaiine 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_1. Kindly add the link to the video record presenting the features of your application, you can either use loom or zoom._
_2. Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (codecaiine) [reply index.html]: I fixed it.

update

File: index.html (hunk 3 above)
Comment (codecaiine) [reply index.html]: footer

File: index.html (hunk 4 above)
Comment (codecaiine) [reply index.html]: nav

File: README.md (hunk 1 above)
Comment (codecaiine) [reply README.md:62]: Ak

Comment (rloterh) [reply README.md:62]: This issue is well resolved! ✔️

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

File: index.html [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+    </section>
+    <section class="section-speaker" id="my_speakers">
+        <div class="container">
+            <div class="main-title">
+                <h1 class="title">Featured Speakers</h1>
+                <div class="title-line"></div>
+            </div>
+
+            <div class="row mb-2" id="speakers-list"></div>
```
Comment (rloterh) [inline index.html:142]: - [ ] OPTIONAL
Also, kindly notice that the images of speakers in the `Featured Speakers` section has a decorated `background images`. Kindly endeavor to improve upon this aspect

**Your design (Featured Speakers section):**
image


**The original design (Featured Speakers section):**
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    </footer>
+
+
+
+
```
Comment (rloterh) [inline about.html]: - [ ] OPTIONAL
To enhance a cleaner code kindly endeavor to remove **all** extra lines which are serving no purpose in your code.

File: README.md [hunk 7]
```diff
@@ -1 +1,68 @@
...
+
+![screenshot](./images/conf-mob.PNG)
+
+ 
+
+## Live Demo
+
+[Live Demo Link](https://codecaiine.github.io/conference-website/)
+
+[Demo video link](https://www.loom.com/share/0031406c6c2b4a87b2066580a29365fb)
```
Comment (rloterh) [inline README.md]: - [ ] Nice job by adding video description of your project as requested by the previous code reviewer. However, your video description **not detailed**. Kindly ensure that video description also covers some salient aspect your **code base** for this project. (Video should be at least 4mins and max of 5 mins)

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Yannick-Noel, 

You did exceptionally great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚆 🚥 

## Good Points 👍  
- Great UI design
- Responsive layout
- No linter errors

## Required Changes ♻️
- [ ] _Kindy check the comments under this reviews_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked [OPTIONAL] under this review:


Cheers..Happy coding..and keep Soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 3 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**